cfme/utils/auth/__init__.py

```diff
@@ -97,7 +98,8 @@ def as_fill_external_value(self):
         Same view for all auth provider types
         """
         class_attrs = [att.name for att in self.__attrs_attrs__]
-        include_attrs = [getattr(self.__class__, name)
```

Comment:
**optional** we seem to have 3 places that based on a name list for fetch attr attributes,
maybe that can be consolidated

Reply:
I'll take another look at consolidating, I had before, but they're all referencing different classes for the included attributes.

Reply:
Was able to collapse a bit of it.